### ISSUE TITLE
Can properties other than values be referenced?

### ISSUE THREAD

**Issue by TrevorBurnham** [design-tokens/community-group#127]
The current spec states that "A design token's *value* can be a reference to another token," and the example shows one token's `$value` referencing another token to inherit its `$value`:

```json
{
  "group name": {
    "token name": {
      "$value": 1234
    }
  },
  "alias name": {
    "$value": "{group name.token name}"
  }
}
```

I think that only allowing value-to-value references is somewhat limiting. The benefits of aliases that are stated in the spec are just as applicable to other properties:

> Aliases are useful for:
>
> * Expressing design choices
> * Eliminating repetition of values in token files (DRYing up the code)

**Comment by c1rrus** [comment]
You're right. As it stands in the current draft, only tokens can be referenced.

What other attributes of tokens do you think would be useful to reference? (Bear in mind that currently there's only `$type`, `$description` and `$extensions`)

Another consideration is the rules around what things should be allowed to be reference what other things. For example, should one token's value be allowed to be a reference to another's description? And how might that affect [the algorithm for determining a token's type](https://tr.designtokens.org/format/#type-0)?

Some concrete use-cases would be helpful to evaluate and discuss this idea further.

**Comment by CITguy** [comment]
@kaelig: did you intend to link back to this issue?

**Comment by kaelig** [comment]
Oops, definitely not! It's been a while so I'm not sure what I had in mind, sorry!

**Comment by Ka77ius** [comment]
@kaelig
